B9PartSwitch/PartSwitch/ModuleB9PartSwitch.cs

```diff
@@ -196,6 +201,39 @@ public void Start()
 
         #endregion
 
+        #region Update
+
+        // We need to give some time to PartModules to properly initialize themself on load before attempting to disable them
+        // From testing, it seems that we need 2 FixedUpdates after the partmodule start for everything to be allright
+        // For example, the part.Effects emitters used by ModuleRCSFX and ModuleEnginesFX are still null on the second FixedUpdate.
+        // This need to be tested in heavy load situation (500+ parts vessel) to ensure that this stays consistent.
+
+        public void FixedUpdate()
```

Comment:
I will try to do that (something new to learn for me), but is it needed ? Since I identified the number of frames that needed skipping in each situations (none in the editor, 2 in flight)...

Reply:
Either will work, I just tend to prefer the coroutine approach since it doesn't require any additional state to be stored, or implementing `FixedUpdate`